website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -3,102 +3,57 @@
 ### When to use
 
 - To display contextual information resulting from a user’s action.
-- To communicate an ongoing process, e.g., “Creating cluster”.
+- To communicate a background process that has either started or ended, e.g., “Creating cluster”.
 
 ### When not to use
 
-- To display persistent messages or promotional content, consider [Alert](/components/alert).
+- To display contextual associative errors (e.g. form validation), or promotional content, consider [Alert](/components/alert).
```

Comment:
How does this differ from "To display contextual information resulting from a user's action." in line 5?

Reply:
I think we can just remove line 5, as line 10 is far more detailed as to when not to use.

Reply:
Line 5 could be expanded to say its ok to use a toast as a save confirmation (like after someone submits a form)

Reply:
It may be redundant because line 6 was meant for that. I'll remove 5 :)